mex/common/wikidata/extract.py

```diff
@@ -54,7 +54,12 @@ def search_organization_by_label(
                 f"IndexError: Error processing results for {item_label}"
             ) from exc
```

Comment:
why are we catching `requests.exceptions` errors **here** and not around `_get_organization_details`?

coverage report already gives a good hint for that issue:

```
Name                                     Stmts   Miss Branch BrPart  Cover   Missing
------------------------------------------------------------------------------------
mex/common/wikidata/extract.py              29      4     10      3    82%   41, 45, 52-53
```

Reply:
but **how** would `wd_item_id = results[0]["item"]["value"].split("/")[-1]` produce an `HTTPError`?

Reply:
thats a good question which I dont know the answer of. Maybe when we were working with generators the call was made when it tried to access something in the `results` list. But now its irrelevant, and I removed it.